crates/transaction-pool/src/traits.rs

```diff
@@ -921,13 +924,13 @@ pub trait PoolTransaction: fmt::Debug + Send + Sync + Clone {
     fn chain_id(&self) -> Option<u64>;
 }
 
-/// An extension trait that provides additional interfaces for the
-/// [`EthTransactionValidator`](crate::EthTransactionValidator).
+/// Super trait for transactions that can be converted to and from Eth transactions
 pub trait EthPoolTransaction:
     PoolTransaction<
-    Pooled = PooledTransactionsElementEcRecovered,
-    Consensus = TransactionSignedEcRecovered,
->
+        Consensus: From<TransactionSignedEcRecovered> + Into<TransactionSignedEcRecovered>,
+        Pooled: From<PooledTransactionsElementEcRecovered>
+                    + Into<PooledTransactionsElementEcRecovered>,
+    > + IntoRecoveredTransaction
```

Comment:
why do we need this now?

Reply:
We don't actually need it. I kept it there because the method from this trait was used and rather than using `.into_consensus.into()` I just left it as is. I can for sure get rid of it though!

Reply:
I'd like to remove this, don't think this is used